app/src/main/java/com/example/seminar1/data/sopt/RequestSignIn.kt

```diff
@@ -0,0 +1,9 @@
+package com.example.seminar1.data.sopt
+
+import com.google.gson.annotations.SerializedName
+
+data class RequestSignIn(
```

Comment:
```suggestion
data class SignInRequest(
```
별 중요한건 아닌데 이렇게 하면 의미가 더 잘 전달될 것 같습니다.